libvmaf/src/output.c

```diff
@@ -99,7 +99,7 @@ int vmaf_write_output_xml(VmafContext *vmaf, VmafFeatureCollector *fc,
             int err = vmaf_feature_score_pooled(vmaf, feature_name, j, &score,
                                                 0, pic_cnt - 1);
             if (!err)
-                fprintf(outfile, "%s=\"%.6f\" ", pool_method_name[j], score);
+                fprintf(outfile, "%s=\"%.30f\" ", pool_method_name[j], score);
```

Comment:
Are these `.30f` formattings deliberately left in or are they holdovers from debugging?

Reply:
This is to take care of ST-RRED accuracy. ST-RRED values go upto E-30